README.md

```diff
@@ -255,6 +259,110 @@ Here is an example:
 }
 ```
 
+#### SPECTER specific parameters (affinity scores):
+- `model_params.specter_dir`: Path to the unpacked SPECTER directory. The model checkpoint will be loaded relative to this directory.
+- `model_params.work_dir`: When running SPECTER, this is where the intermediate files are stored.
+- `model_params.use_cuda`: Boolean to indicate whether to use GPU (`true`) or CPU (`false`) when running SPECTER. Currently, only 1 GPU is supported, but there does not seem to be necessary to have more.
+- `model_params.batch_size`: Batch size when running SPECTER. This defaults to 16.
+- `model_params.publications_path`: When running SPECTER, this is where the embedded abstracts/titles of the Reviewers (and Area Chairs) are stored.
+- `model_params.submissions_path`: When running SPECTER, this is where the embedded abstracts/titles of the Submissions are stored.
+- `model_params.average_score` (boolean, defaults to `false`): This parameter specifies that the reviewer is assigned based on the average similarity of the submission to the authored publication embeddings. Exactly one of `model_params.average_score` and `model_params.max_score` must be `true`.
+- `model_params.max_score` (boolean, defaults to `true`): This parameter specifies that the reviewer is assigned based on the max similarity of the submission to the authored publication embeddings. Exactly one of `model_params.average_score` and `model_params.max_score` must be `true`.
+- `model_params.skip_specter`: Since running SPECTER can take a significant amount of time, the vectors are saved in `model_params.submissions_path` and `model_params.publications_path`. The jsonl files will be loaded with all the vectors.
+
+Here is an example:
+```
+{
+    "name": "iclr2020_specter",
+    "dataset": {
+        "directory": "./"
+    },
+    "model": "specter",
+    "model_params": {
+        "specter_dir": "../specter/",
+        "work_dir": "./",
+        "average_score": false,
+        "max_score": true,
+        "use_cuda": true,
+        "batch_size": 16,
+        "publications_path": "./",
```

Comment:
can we use `dataset` property and remove these two: `publications_path` and `submissions_path`?

Reply:
never mind, this seems to be something else, I'm not sure why we need different paths.